src/pages/login.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { Navigate } from 'react-router-dom';
+import { loginUser } from '../util/loginUser';
+import './loginStyle.css';
+
+const Login = () => {
+  const [isbuttonDisabled, setIsButtonDisabled] = useState(false);
+  const [textUserEmail, setTextUserEmail] = useState('');
+  const [displayErrorMessage, setDisplayErrorMessage] = useState(false);
+  const [isLogin, setIsLogin] = useState(false);
+  useEffect(() => {
+    if (/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,4})+$/.test(textUserEmail)) {
+      setIsButtonDisabled(false);
+    } else {
+      setIsButtonDisabled(true);
+    }
+  }, [textUserEmail]);
+
+  const handleTextChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTextUserEmail(event.target.value);
+  };
+  const handleClick = async () => {
+    const isExisting = await loginUser(textUserEmail);
+    if (isExisting) {
+      setIsLogin(true);

```

Comment:
This is a little confusing here. It's not immediately clear what this is actually for. Rather than setting some state and rendering a navigate to redirect you should just route to the next screen right here.

Reply:
Yes i was trying to redirect in function handleClick but i forgot to use useNavigate and just redirect, my bad.